core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -95,6 +97,30 @@ class ShuffleDependency[K: ClassTag, V: ClassTag, C: ClassTag](
   val shuffleHandle: ShuffleHandle = _rdd.context.env.shuffleManager.registerShuffle(
     shuffleId, this)
 
+  // By default, shuffle merge is enabled for ShuffleDependency if push based shuffle is enabled
+  private[spark] var _shuffleMergeEnabled =
```

Comment:
Why does this variable name start with `_`? 
Other variables in this class don't follow this.

Reply:
`_` is used so that we can distinguish between class variables and the method arguments as all these are just some getters and setters. This is not the case with other variables. If not we have to change the name to something different that would make it difficult quickly inspect the code.

Reply:
Removed the usages of `_`